scoper.inc.php

```diff
@@ -110,6 +116,10 @@ function( $file_path, $prefix, $contents ) {
 				$contents = str_replace( "'Google_", "'" . $prefix . '\Google_', $contents );
 				$contents = str_replace( '"Google_', '"' . $prefix . '\Google_', $contents );
 			}
+			if ( false !== strpos( $file_path, 'apiclient-services-subscribewithgoogle' ) ) {
+				// Rewrite "Class_Name" to Class_Name::class to inherit namespace.
+				$contents = preg_replace( '/"(Google_[^"]+)"/', '\\1::class', $contents );
```

Comment:
Without this, a fatal error will be raised in the result of a successful request because the Google collection will try to map the response into the respective models but using the wrong (unprefixed) class name, which won't exist. This was originally missed because the API wasn't working yet 😄